src/Widgets/TopNavigationWidget/SubComponents/LanguageSwitch.tsx

```diff
@@ -65,17 +88,18 @@ export const LanguageSwitch = connect(function LanguageSwitch({
 })
 
 const LanguageLabel = connect(function LanguageLabel({
+  label,
+  language,
   root,
   showTextLabel,
 }: {
+  label: string
+  language: string | undefined
```

Comment:
* [ ] FYI Passing in the language blurs the source of truth. Since `root` is mandatory, and we already get the `siteLanguageIcon` from there, we should also use `root().language()`.
* [ ] FYI I see how recycling `label` makes things cheaper and also a label is not necessarily tied to the site, so passing in `label` feels more acceptable than `language`.